index.html

```diff
@@ -86,7 +86,7 @@ <h2 class="section-title" id="about">about.</h2>
         <h2 class="section-title" id="skills">Skills.</h2>
         <ul>
           <li>JavaScript</li>
-          <li>Vue</li>
```

Comment:
The changes are great but I rec0mmend you to add Express and React Native!!! and also fix your experience and the years.